app/models/patron_request.rb

```diff
@@ -169,7 +169,9 @@ def item_title
   # @return [Array<Folio::Item>] the items in the origin location
   def items_in_location
     @items_in_location ||= bib_data.items.select do |item|
-      if item.effective_location.details['searchworksTreatTemporaryLocationAsPermanentLocation'] == 'true'
+      if barcodes.length > 0
+        barcodes.include?(item.barcode)
```

Comment:
```suggestion
        item.barcode.in?(barcodes) || item.id.in?(barcodes)
```

We do this in `selected_items` to handle items without barcodes. Presumably we should be consistent?

Reply:
makes sense.